modules/mobile/spec/request/labs_and_tests_request_spec.rb

```diff
@@ -3,19 +3,18 @@
 require 'rails_helper'
 require 'support/mr_client_helpers'
 require 'medical_records/client'
-require_relative '../support/helpers/iam_session_helper'
+require_relative '../support/helpers/sis_session_helper'
 
 RSpec.describe 'labs and tests', type: :request do
   include MedicalRecords::ClientHelpers
   include SchemaMatchers
 
+  let(:user) { sis_user }
   let(:user_id) { '32000225' }
-  let(:va_patient) { true }
-  let(:current_user) { build(:user, :mhv, va_patient:, mhv_account_type:) }
 
   before do
     allow(MedicalRecords::Client).to receive(:new).and_return(authenticated_client)
-    sign_in_as(current_user)
+    sign_in_as(user)
     allow_any_instance_of(IAMUser).to receive(:icn).and_return(user_id)
```

Comment:
This should also not be necessary. You can just do `let!(:user) { sis_user(icn: '32000225') }`